### PR TITLE
add events to protobuf

### PR DESCRIPTION
**Pull request by mdubbyap**

### CODE REVIEW

**Comment by mdubbyap** [comment]
@jloomis i'm adding proper event support to ingest, which means it makes sense to do protobuf now as well (i think that was somewhere on your distant todo list). So far this is working for me, let me know if i screwed anything up.

File: signalfx-protoc/src/main/protobuf/signal_fx_protocol_buffers.proto [hunk 1]
```diff
@@ -49,3 +48,61 @@ message PointValue {
...
 }
+
+/**
+* Different categories of events supported
+*/
+enum EventCategory {
+    /**
+     * Created by user via UI or API, e.g. a deployment event
+     */
+    USER_DEFINED = 1;
```
Comment (jloomis) [inline signalfx-protoc/src/main/protobuf/signal_fx_protocol_buffers.proto]: make values match what we have in thrift? (ALERT = 100000 etc)

**Comment by mdubbyap** [comment]
Updated.

**Comment by mdubbyap** [comment]
Also note, i've added the exception category for @philatsf 's log appender.  I'll update the thrift file soon as well.

**Pull request merged**